rulings/cards-rulings.yaml

```
@@ -3089,6 +3089,9 @@
 102208|Zip Gun:
   - Damage done to the bearer is environmental. [LSJ 19970801]
   - No damage is done to the bearer if he does not strike with the gun (eg. if it is destroyed with first strike). [LSJ 20100310]
+102223|Form of the Bat:
+  - "[pro], [PRO] The maneuver (and the ban on equipments) can only be used if the block is successful, and only in the resulting (block-induced) combat. It does not carry over to a follow-up combat (eg. [CEL] {Psyche!}). [LSJ 20010813] [LSJ 20010819-2]"
+  - "[pro], [PRO] The maneuver (and the ban on equipments) is provided again if a second block happens on the same action (eg. after [PRO] {Form of Mist}). [LSJ 20010814-2]"
```

When the ruling applies to _all_ discipline levels, better not use any discipline prefix on the ruling line.
```suggestion
  - "The maneuver, and equipment restriction, apply only in the combat resulting from a successful block. They do not carry over to follow-up combats (eg. [CEL] {Psyche!}). [LSJ 20010813] [LSJ 20010819-2]"
  - "The maneuver, and equipment restriction, apply again if a second block happens on the same action (eg. after [PRO] {Form of Mist}). [LSJ 20010814-2]"
```